app/routes/program-year-visualizations.js

```diff
@@ -1,17 +1,25 @@
 import { inject as service } from '@ember/service';
 import Route from '@ember/routing/route';
-import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 
-export default Route.extend(AuthenticatedRouteMixin, {
-  store: service(),
+export default class ProgramYearVisualizationsRoute extends Route {
+  @service store;
+  @service session;
+
   // eslint-disable-next-line ember/avoid-leaking-state-in-ember-objects
```

Comment:
I'm betting this eslint modifier can go away now that this is a class.

Reply:
:heavy_check_mark: